content/id/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -0,0 +1,632 @@
+tersebut secara _asynchrounous_ oleh Kubelet. Perlu dicatat bahwa karena scheduler
+menggunakan nilai `status.allocatable` milik Node saat mengevaluasi muat atau tidaknya
+Pod, mungkin ada waktu tunda pendek antara melakukan `PATCH` terhadap kapasitas Node
+dengan sumber daya baru dengan Pod pertama yang meminta sumber daya tersebut untuk
+dapat dijadwalkan pada Node tersebut.
+
+**Contoh:**
+
+Berikut sebuah contoh yang menunjukkan bagaimana cara menggunakan `curl` untuk
+mengirim permintaan HTTP yang menyatakan lima sumber daya "example.com/foo" pada
+Node `k8s-node-1` yang memiliki master `k8s-master`.
+
+```shell
+curl --header "Content-Type: application/json-patch+json" \
+--request PATCH \
+--data '[{"op": "add", "path": "/status/capacity/example.com~1foo", "value": "5"}]' \
+http://k8s-master:8080/api/v1/nodes/k8s-node-1/status
+```
+
+{{< note >}}
+Pada permintaan HTTP di atas, `~1` adalah _encoding_ untuk karakter `/` pada jalur (_path_) _patch_.
+Nilai jalur operasi tersebut di dalam JSON-Patch diinterpretasikan sebagai sebuah JSON-Pointer.
+Untuk lebih lanjut, lihat [IETF RFC 6901, bagian 3](https://tools.ietf.org/html/rfc6901#section-3).
+{{< /note >}}
+
+#### Sumber daya yang diperluas pada tingkat klaster
+
+Sumber daya yang diperluas pada tingkat klaster tidak terikat pada Node. Mereka
+biasanya dikelola oleh _scheduler extender_, yang menangani penggunaan sumber daya
+dan kuota sumber daya.
+
+Kamu dapat memerinci sumber daya yang diperluas yang ditangani oleh _scheduler extender_
+pada [konfigurasi kebijakan scheduler](https://github.com/kubernetes/kubernetes/blob/release-1.10/pkg/scheduler/api/v1/types.go#L31).
+
+**Contoh:**
+
+Konfigurasi untuk sebuah kebijakan scheduler berikut menunjukkan bahwa
+sumber daya yang diperluas pada tingkat klaster "example.com/foo" ditangani
+oleh _scheduler extender_.
+
+- Scheduler mengirim sebuah Pod ke _scheduler extender_ hanya jika Pod tersebut
+    meminta "example.com/foo".
+- Kolom `ignoredByScheduler` memerinci bahwa scheduler tidak memeriksa sumber daya
+    "example.com/foo" pada predikat `PodFitsResources` miliknya.
+
+```json
+{
+  "kind": "Policy",
+  "apiVersion": "v1",
+  "extenders": [
+    {
+      "urlPrefix":"<extender-endpoint>",
+      "bindVerb": "bind",
+      "managedResources": [
+        {
+          "name": "example.com/foo",
+          "ignoredByScheduler": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Menggunakan sumber daya yang diperluas
+
+Pengguna dapat menggunakan sumber daya yang diperluas di dalam spesifikasi Pod
+seperti CPU dan memori. Scheduler menangani akuntansi sumber daya tersebut agar
+tidak ada alokasi untuk yang melebihi jumlah yang tersedia.
+
+API server membatasi jumlah sumber daya yang diperluas dalam bentuk
+bilangan bulat. Contoh jumlah yang _valid_ adalah `3`, `3000m`, dan
+`3Ki`. Contoh jumlah yang _tidak valid_ adalah `0.5` dan `1500m`.
+
+{{< note >}}
+Sumber daya yang diperluas menggantikan Opaque Integer Resource.
```

Comment:
Sumber daya yang diperluas menggantikan _Opaque Integer Resource_.

Reply:
I believe Opaque Integer Resource refers to an old API object (now removed), so I didn't italicize it. What do you think? @danninov

Reply:
It was API object, but already replaced with Extended resources